R/get_data.R

```diff
@@ -22,9 +22,6 @@
 #' `options("silence_named_get_data_warning" = TRUE)` - which you can set
 #' in your .Rprofile file so the option persists across sessions.
 #'
-#' @param format Defaults to NULL which will first check to see if the record is a WMS/WFS record.
-#' If so an sf object is returned. Otherwise a format needs to be specified. `bcdc_get_record` can
-#' be used to identify which formats are available.
 #' @param resource option argument used when there are multiple data files of the same format
 #' within the same record. See examples.
 #' @param ... arguments passed to other functions. For spatial (WMS/WFS) data these are passed to
```

Comment:
Here I think we could provide explicit information on the functions used to read the various formats so users know what they can pass to `...`.

Reply:
I ended exporting a new function here. `bcdc_read_functions` which gives a user the function that is used to read in. Then in the examples for `bcdc_get_data` I outlined a very simple workflow of how one would use it. I think that main utility of using the read in functions arguments is to smooth the data pipeline. Some initial setup will always be required. Implemented here: https://github.com/bcgov/bcdata/commit/c2d4db4731d553d9151f3475e099021421b92774

Reply:
Nice! Could we also print a message saying `"Importing dataset using package::function()"` or is that overkill?

Reply:
Also, great example 👍

Reply:
Also see #72